COPYRIGHT

```diff
@@ -17,3 +17,9 @@ operating system images.
 Contains modified hyper-proxy files [mod.rs, stream.rs, tunnel.rs] from
 https://github.com/tafia/hyper-proxy 2021-09-20.
 Copyright (c) 2017 Johann Tuffe. Licensed under the MIT License.
+
+=^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=   =^..^=
+
+Contains modified aws-smithy-experimental files [README.md, Cargo.toml] from
```

Comment:
I would drop this specific list and update this in a PR that actually adds source code changes:
```
Contains aws-smithy-experimental files from
```